R/auditCommonFunctions.R

```diff
@@ -3100,27 +3100,23 @@ gettextf <- function(fmt, ..., domain = NULL) {
         values <- values * prevOptions[["N.units"]]
       }
 
-      yBreaks <- jaspGraphs::getPrettyAxisBreaks(c(0, values), min.n = 4)
-
-      if (options[["display"]] == "amount") {
-        x.labels <- format(jaspGraphs::getPrettyAxisBreaks(seq(0, 1.1 * max(values), length.out = 100), min.n = 4), scientific = FALSE)
-        values.labels <- ceiling(values)
-      } else {
-        x.labels <- paste0(round(jaspGraphs::getPrettyAxisBreaks(seq(0, 1.1 * max(values), length.out = 100), min.n = 4) * 100, 4), "%")
-        values.labels <- paste0(round(values * 100, 2), "%")
-      }
-
       plotData <- data.frame(x = label, y = values)
       plotData$x <- factor(plotData$x, levels = plotData$x)
 
-      yLimits <- c(0, 1.1 * max(values))
-      yBreaks <- jaspGraphs::getPrettyAxisBreaks(seq(0, 1.1 * max(values), length.out = 100), min.n = 4)
+      if (all(plotData$y > 0)) {
+        yBreaks <- jaspGraphs::getPrettyAxisBreaks(c(0, 1.2 * max(values)), min.n = 4)
+        yLimits <- c(0, max(yBreaks))
+      } else {
+        yBreaks <- jaspGraphs::getPrettyAxisBreaks(c(min(values), 1.2 * max(values)), min.n = 4)
```

Comment:
I am wondering whether it would be cleaner to just use `expand` argument in `ggplot2::scale_y_continuous()` a little bit down in the code, as this is exactly the situation this argument is there for. I'll leave it up to you whether you want to do it.

Reply:
I'll leave this for now.